dashboard/src/components/Header.vue

```diff
@@ -62,4 +62,9 @@ export default {
 };
 </script>
 
-<style></style>
+<style lang="scss">
+	nav {
+		padding: 0 1.5em;
+		border-radius: 0em !important;
```

Comment:
No clue where navbar gets a border radius from. Navbars are flat if they span across the entire width.

Reply:
/shrug source could be any of bulma, buefy or bulmaswatch

Reply:
nice catch anyways it was annoying me as well